cogs/tmpc.py

```diff
@@ -579,6 +541,34 @@ async def check_tmpc_channel(self, member: Member, _id: int, is_mod: bool = Fals
             return document
         raise NotOwnerError(is_mod=is_mod, owner=document.owner.mention)
 
+    async def get_temp_channel(self, ctx: Context) -> TempChannel:
+
+        channels: list[TempChannel] = await self.channel_db.find({DBKeyWrapperEnum.OWNER.value: ctx.author.id})
+        if not channels:
+            raise YouOwnNoChannelsError()
+        elif len(channels) == 1:
+            return channels.pop()
+
+        pool = {channel.chat.name: channel for channel in channels}
+
+        options = Select(
+            placeholder="Select your group",
+            options=[SelectOption(label=channel.chat.name, value=channel.chat.name) for channel in channels])
```

Comment:
what if there are multiple channels names the same?
Idea: prefix with category

Reply:
Channels with the same name is solved somewhere else.

Reply:
Oh yeah right seen that change